### ISSUE TITLE
private method `set_output' called for #<LiveKit::EgressServiceClient:0x000055f15cca5cf8>

### ISSUE THREAD

**Issue by izinko** [livekit/server-sdk-ruby#39]
private method `set_output' called for #<LiveKit::EgressServiceClient:0x000055f15cca5cf8>
.rvm/gems/ruby-2.6.1@immerss-portal/gems/livekit-server-sdk-0.6.1/lib/livekit/egress_service_client.rb:42:in `start_room_composite_egress'

I think in this case it must use set_output(request, output) instead self.set_output(request, output) in whole lib/livekit/egress_service_client.rb file

**Comment by davidzhao** [comment]
You are right. That looks like a bug.. would you like to create a PR for it?
